docs/announce/6.e.md

```diff
@@ -0,0 +1,70 @@
+
+# Announce: Perl 6.e Language Specification Release
+
+_This to be filled up upon actual release_
+
+## Authors
+
+The following people contributed to this version of the language,
+including making documentation updates and contributing work to known
+open‑source compiler implementations. The list is ordered alphabetically.
+
+If you believe your name has been erroneously omited, please contact us
+(https://perl6.org/irc), and we'll update the primary copy of this list.
+
+_This to be filled up upon actual release_
+
+## Changelog
+
+This document lists changes in Perl 6.e language
+from Perl 6.d version. A particular implementation of the language
+may contain additional changes; please consult with the changelog for your
+implementation.
+
+At the same time, a particular implementation may have had certain features
+already implemented during the previous versions period. This ChangeLog concerns
+itself with new features added to the specification on a language level and not
+the status of their implementation in a particular compiler.
+
+### Scope / Target Audience
+
+This ChangeLog is targeted towards language users, to help with preparation to use
+compilers supporting latest language version. Thus, it does not contain every minute
+change to the specification that occurred. Implementations wishing to ensure full
+compliance with the new version of the language specification should execute the
+test suite available at https://github.com/perl6/roast/ and examine any failing tests.
+
+There are new features that did not exist in the previous releases. For full details about them,
+please consult with the language documentation on https://docs.perl6.org/
+Items in *Version-Controlled Changes* section are protected by version pragma and older
+behaviours can be obtained by explicitly using `use v6.<rev>` to request an older language
+version. All other changes do not conflict with the previous revisions and implementations
+may choose to make them available even when an earlier language version is requested.
+
+#### Version-Controlled Changes
+
+_Fill in this section as soon as such changes appear. The following line is a sample from 6.d announce_
+
+- **[6.d]** `&await` no longer blocks *a thread* while waiting
+
+_The following sections are to be filled with upcoming changes in 6.e language_
+
+#### Deprecations
+
+These methods are deprecated in 6.e language and will be removed in 6.f.
+Implementations may choose to emit deprecation warnings or to offer these
+methods for a longer period than 6.f release.
+
+#### New Behaviors
+
+#### New Parameters and Arguments
+
+#### New Routines and Operators
+
+#### New Types
+
+#### New Variables
+
+#### Clarifications of Edge Case/Coercion Behaviour
```

Comment:
Would this include disambiguation of behavior? If not, maybe a new subsection would be in order.

Reply:
What to include into the announcement would be absolutely up to the release manager. I created a template based on `6.d` announce. The purpose of the template is to fill it in as soon as there is something to be noted in it.